config/webpack.config.prod.js

```diff
@@ -397,9 +397,7 @@ module.exports = {
     new webpack.NormalModuleReplacementPlugin(
       /.*\.sol/,
       function(resource) {
-        const targetNetwork = process.env.TARGET_NETWORK;
-        const artifactsPath = path.join(paths.appContractArtifacts,targetNetwork);
-        resource.request = resource.request.replace(/.*contracts/,artifactsPath).replace('.sol','.json');
+        resource.request = resource.request.replace(/.*contracts/,paths.appContractArtifacts).replace('.sol','.json');
```

Comment:
@lalexgap might be more readable with commas separating the arguments
```suggestion
        resource.request = resource.request.replace(/.*contracts/, paths.appContractArtifacts).replace('.sol', '.json');
```